src/net.rs

```diff
@@ -270,6 +270,7 @@ impl RedisManager {
                     let conn = client.get_connection()?;
                     thread::sleep(Duration::from_millis(500));
 
+                    conn.set_write_timeout(Some(Duration::from_secs(5)))?;
```

Comment:
Do you need to set `set_read_timeout` too?

Reply:
No, this connection is only for sending message.

Reply:
yes. What I mean is that it is need to set set_read_timeout elsewhere to receive messages from Redis.

Reply:
It is because receiving message connection doesn't have timeout. It's always waiting messages.